src/memefactory/shared/utils.cljs

```diff
@@ -64,3 +65,9 @@
                   (clj->js [content])
                   (clj->js {:type (or mime-type (str "application/json;charset=UTF-8"))}))
              filename))
+
+
+(defn extend-promises-as-channels! []
```

Comment:
This also should be in https://github.com/district0x/async-helpers

Reply:
+1